<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>961</width>
     <height>661</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>961</width>
     <height>661</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>961</width>
     <height>661</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>981</width>
     <height>671</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/img.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>270</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>180</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>270</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outside1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outside2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outside3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outside4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outside5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inside1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inside2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inside3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inside4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inside5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="inside">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>136</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>176</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>216</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>256</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>295</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tech">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>460</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(225, 205, 191);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tech_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(225, 205, 191);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(0, 231, 146);
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color:rgb(62, 0, 93);
}
</string>
   </property>
   <property name="text">
    <string>Тех.Служба</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>530</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(111, 222, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
